matlab/schemas/+stimulus/Clip.m

```diff
@@ -0,0 +1,61 @@
+%{
+# Movie clip condition
+-> stimulus.Condition
+-----
+-> stimulus.MovieClip
+cut_after              : float           # (s) cuts off after this duration
+%}
+
+classdef Clip < dj.Manual & stimulus.core.Visual
+    
+    properties(Constant)
+        variation = '1'
+        movie_dir = '/Users/dimitri/stimuli'
```

Comment:
In +stimulus/MovieClip.m you put movie_dir='movies', maybe movie_dir='stimuli' is good here